lib/apipie/application.rb

```diff
@@ -380,7 +380,7 @@ def active_dsl?
 
     # @deprecated Use {#get_resource_id} instead
     def get_resource_name(klass)
-      ActiveSupport::Deprecation.warn(
+      ActiveSupport::Deprecation.new.warn(
```

Comment:
Reading https://api.rubyonrails.org/classes/ActiveSupport/Deprecation.html I think you should really set a specific deprecation horizon. So something like:
```suggestion
      ActiveSupport::Deprecation.new('2.0', 'apipie-rails').warn(
```